src/main/java/README.md

```diff
@@ -0,0 +1,28 @@
+## Additional Setup for Client-side to CPK-N Code Samples
+
+#### KeyVaultClientEncryption 
```

Comment:
Can you add more information about set up here? Like user needs Java 8 installed, etc

Reply:
Most of our customers use Java 8 so we should make sure the program works with at least Java 8